wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -44,13 +44,31 @@
 #include <esp_types.h>
 #include <esp_log.h>
 
+#ifndef _INTPTR_T_DECLARED
```

Comment:
These normally come from stdint.h. Any reason not to use it?

Reply:
I should have left myself a reminder; I don't recall. You're correct. For the ESP32 ESP-IDF, this is defined in:

`xtensa-esp-elf\xtensa-esp-elf\sys-include\sys\_stdint.h`

The missing `_INTPTR_T_DECLARED` was encountered in one of the other environments such as Arduino, PlatformIO, possibly the ESP8266 RTOS SDK.

There's probably no harm in leaving this, but I can put it in a separate PR once I re-discover why I needed it.

Reply:
Fine to leave it as-is.